### ISSUE TITLE
Maximum parallel repairs reached, postponing repair run

### ISSUE THREAD

**Issue by mortenjoenby** [thelastpickle/cassandra-reaper#1219]
Reaper: v2.3.0, backend: postgres
Cassandra v3.11.4
Cluster of 5 nodes, num_tokens=16, initial_tokens specifying 16 tokens per node.

I have had some issues with Reaper not starting repairs due to this message - shown in Reaper Admin UI (not in the logs!):
"Maximum parallel repairs reached, postponing repair run"

I did the following to make sure I started on a clean slate:
1) Delete all schedules
2) Stop Reaper service
3) Rolling restart of Cassandra nodes
4) Start Reaper service

All schedules were re-created using autoScheduler.
12 hours between schedules
repairIntensity: 0.2
segmentCountPerNode: 16

I have no idea what repairs could be running preventing the Reaper schedule from starting.

┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1736) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-1736
┆priority: Medium

**Comment by mortenjoenby** [comment]
I restarted the Reaper service after having waited for ~3 hours. Then the repair started immediately.

**Comment by mortenjoenby** [comment]
Now that Reaper finished, the same problem happens again for the next scheduled run, meaning that it never starts.

**Comment by ReillyBrogan** [comment]
Your reaper version is more than a year out of date, your cassandra version more than three (!). There have been a substantial number of bugfixes to both of those, I'd recommend updating both as chances are very high that your issue has already been identified and fixed.

**Comment by mortenjoenby** [comment]
Hi @ReillyBrogan .
You are 100% correct :) We will be upgrading both, but not until later this year.
We managed to find the problem though, which was some "old" repair jobs in PAUSED state that had survived me deleteing all schedules to re-create them with other schedule intervals. I had 2 of those blocking new repairs.
I was able to delete them using 'spreaper' and now everything is running smoothly.
Thanks for your feedback! 
.... Upgrade it is ... 👍
